### PR TITLE
DM-35670: Remove gen2 config tests.

### PR DESCRIPTION
**Pull request by erykoff**

### CODE REVIEW

File: tests/test_ingest.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 from lsst.daf.butler import Butler, DataCoordinate
 from lsst.obs.base.ingest_tests import IngestTestBase
+import lsst.afw.cameraGeom.testUtils
```
Comment (timj) [inline tests/test_ingest.py:31]: This seems wrong to me. This file does not call `assertDetectorsEqual`. This line should be added instead to `obs_base/python/lsst/obs/base/ingest_tests.py`.

**Review by timj — CHANGES_REQUESTED** [review]
I think the testUtils fix is the wrong one and needs to be done in obs_base.

For the test_config.py I had no idea what it was really trying to achieve. Was it trying to see that loading config files would load? If other obs packages don't test that then it seems odd that this one is.

Please remove the processCcd.py and singleFrameDriver.py config files -- they are no longer used.

**Comment by erykoff** [comment]
I also couldn't figure out what test_config was trying to do.  I hope it's okay to remove it?

**Review by timj — APPROVED** [review]
Looks good. It seems that the config test is just loading configs -- maybe that's entirely fine but isn't something we do elsewhere.

**Pull request merged**